lib/hammer_cli_foreman/hostgroup.rb

```diff
@@ -190,6 +190,15 @@ def validate_options
       end
     end
 
+    class RebuildConfigCommand < HammerCLIForeman::SingleResourceCommand
+      action :rebuild_config
+      command_name "rebuild-config"
+      desc _('Rebuild orchestration related configurations for hostgroup')
+      success_message _('Configuration successfully rebuilt.')
+
```

Comment:
We usually require also failure_message. Is it missing on purpose?

Reply:
Not on purpose, just how this particular subcommand is defined for hosts which is where I copied the code from.

Reply:
Added `failure_message`